Model/Reclaim.php

```diff
@@ -235,4 +295,9 @@ public function _getImages($product){
         }
         return $image_array;
     }
+    public function _getStockItem($productId)
+    {
+        $stock = $this->_stockItemRepository->get($productId);
+        return $stock->getManageStock();
```

Comment:
do you need the var to return vs just retuning the `this->_stockItemRepository->($productId);`